src/kiwixapp.cpp

```diff
@@ -403,8 +403,7 @@ void KiwixApp::createActions()
     HIDE_ACTION(SavePageAsAction);
     */
 
-    CREATE_ACTION_SHORTCUT(SearchArticleAction, gt("search-article"), QKeySequence(Qt::CTRL | Qt::Key_L));
-    HIDE_ACTION(SearchArticleAction);
+    CREATE_ACTION_SHORTCUTS(SearchArticleAction, gt("search-article"), QList<QKeySequence>({QKeySequence(Qt::Key_F6), QKeySequence(Qt::CTRL | Qt::Key_L), QKeySequence(Qt::ALT | Qt::Key_D)}));
 
     CREATE_ACTION_SHORTCUT(SearchLibraryAction, gt("search-in-library"), QKeySequence(Qt::CTRL | Qt::SHIFT | Qt::Key_R));
     HIDE_ACTION(SearchLibraryAction);
```

Comment:
Question to @kelson42 and @mgautierfr: `SearchLibraryAction` doesn't seem to work in the current version of `kiwix-desktop`. Did it ever work, was support for it deliberately dropped or was that functionally accidentally broken?

Reply:
@veloman-yunkan I'm not able to answer at the code level. What is user visible impact?

Reply:
@kelson42 The question is not about the change. It is about the search-in-library action that is supposed to be activated via the CTRL+SHIFT+R keyboard shortcut.

Reply:
Probably never worked. Multi-zim search is implemented in libzim but I don't recall we have implemented it in kiwix-desktop.

Reply:
For me this ticket is not about the library... so I wonder why we talk about it! Honestly I have not clue what this PR should fixed because I don't understand the issue in a first place. So probably not going to merge it as long as the issue itself is not "cristal clear".